### PR TITLE
[Added]: Test cases for MultiColumnStatusTextGraph

### PR DESCRIPTION
**Pull request by nxanil**
Added test cases for `MultiColumnStatusText Graph`.

 PASS  src/lib/vis-graphs/Graphs/MultiColumnStatusTextGraph/multiColumnStatusTextGraph.spec.js
 MultiColumnStatusText Graph
   ✓ columnStatusTextGraph Count (14ms)
   ✓ First Data (5ms)
   ✓ Second Data (3ms)
   ✓ Third Data (3ms)
   ✓ First Data Color (2ms)
   ✓ Second Data Color (2ms)
   ✓ Third Data Color (1ms)

Test Suites: 1 passed, 1 total
Tests:       7 passed, 7 total
Snapshots:   0 total
Time:        2.654s

### CODE REVIEW

**Comment by nxanil** [comment]
@preshitg 

can you please review this PR.

File: Graphs/MultiColumnStatusTextGraph/index.js [hunk 1]
```diff
@@ -29,7 +29,9 @@ class MultiColumnStatusTextGraph extends TextGraph {
     renderRow = ({data, column, text, color, fontColor, inline, key}) => (
         <div key={key} style={inline ? {} : {display: "block", marginTop: '2%'}}>
             <div style={{width: "25%", display: "inline-block"}}>
-                <div style={{
+                <div 
+                id={`color-${key}`}
```
Comment (preshitg) [inline Graphs/MultiColumnStatusTextGraph/index.js:33]: Are you adding id to this element only to use it in test cases? 
Is there any alternate method without modifying the source code?

Comment (nxanil) [reply Graphs/MultiColumnStatusTextGraph/index.js]: Changes done.

**Comment by nxanil** [comment]
@preshitg

Feedback changes done. Can you please review this PR.

File: Graphs/testHelper.js [hunk 2]
```diff
@@ -127,6 +127,39 @@ export const appendChildToElement = (element) => document.body.appendChild(eleme
 
 export const removeElement = (element) => document.body.removeChild(element);
 
+export const columnStatusTextGraphData = ($, rowNo) => {
```
Comment (preshitg) [inline Graphs/testHelper.js:130]: Refactor this to receive the textData object and then process further. Sending hard-coded indices doesn't make sense. The test should determine the textData object.

```
export const columnStatusTextGraphData = ($, textData) => {
const value = textData.find('span').map(
        function (i) {
            return $(this).text().trim();
        }
    ).get();
    return value.reverse();
}
```

From your test, compute current textData:
```
let textData = $('.inline-alarms').children().first(); // .next() for further nodes.
const value = columnStatusTextGraphData($, textData);
```

File: Graphs/testHelper.js [hunk 3]
```diff
@@ -127,6 +127,39 @@ export const appendChildToElement = (element) => document.body.appendChild(eleme
...
+        function (i) {
+            return $(this).text().trim();
+        }
+    ).get();
+    return value.reverse();
+}
+
+
+export const getColor = ($, index) => {
+    let currentChild = $('.inline-alarms').children().first();
```
Comment (preshitg) [inline Graphs/testHelper.js:146]: Same as above.

File: Graphs/testHelper.js (hunk 2 above)
Comment (nxanil) [reply Graphs/testHelper.js]: Done.

File: Graphs/testHelper.js (hunk 3 above)
Comment (nxanil) [reply Graphs/testHelper.js]: Done.

**Comment by nxanil** [comment]
@preshitg

Feedback changes done. Can you please review this PR.

**Review by preshitg — APPROVED** [review]

**Pull request merged**